docs/content/Configuration/Databases/SingleStore.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: SingleStore
+permalink: /config/databases/singlestore
+---
+
```

Comment:
Can we add a banner here saying the SingleStore driver isn't supported by us? 

Here's one from the Firebolt page as an example:

```

<WarningBox>

The driver for Firebolt is supported by its vendor. Please report any issues to
their [Help Center][firebolt-help].

</WarningBox>
```

Reply:
Moved the MySQL notice to the top of the page for better visibility.